[2025-01-31T23:07:55.874+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-31T23:07:55.894+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pull_gazettes_data.init_folders_task scheduled__2025-01-30T00:00:00+00:00 [queued]>
[2025-01-31T23:07:55.904+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pull_gazettes_data.init_folders_task scheduled__2025-01-30T00:00:00+00:00 [queued]>
[2025-01-31T23:07:55.904+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2025-01-31T23:07:55.918+0000] {taskinstance.py:2888} INFO - Executing <Task(InitFoldersTask): init_folders_task> on 2025-01-30 00:00:00+00:00
[2025-01-31T23:07:55.931+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=134) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-31T23:07:55.937+0000] {standard_task_runner.py:72} INFO - Started process 141 to run task
[2025-01-31T23:07:55.939+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'pull_gazettes_data', 'init_folders_task', 'scheduled__2025-01-30T00:00:00+00:00', '--job-id', '275', '--raw', '--subdir', 'DAGS_FOLDER/dag_script.py', '--cfg-path', '/tmp/tmpg0ey1_0m']
[2025-01-31T23:07:55.947+0000] {standard_task_runner.py:105} INFO - Job 275: Subtask init_folders_task
[2025-01-31T23:07:56.021+0000] {task_command.py:467} INFO - Running <TaskInstance: pull_gazettes_data.init_folders_task scheduled__2025-01-30T00:00:00+00:00 [running]> on host 41bcec3d818b
[2025-01-31T23:07:56.113+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='***@example.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='pull_gazettes_data' AIRFLOW_CTX_TASK_ID='init_folders_task' AIRFLOW_CTX_EXECUTION_DATE='2025-01-30T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-30T00:00:00+00:00'
[2025-01-31T23:07:56.115+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-31T23:07:56.143+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-31T23:07:56.144+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=pull_gazettes_data, task_id=init_folders_task, run_id=scheduled__2025-01-30T00:00:00+00:00, execution_date=20250130T000000, start_date=20250131T230755, end_date=20250131T230756
[2025-01-31T23:07:56.197+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-31T23:07:56.225+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-31T23:07:56.227+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
